packages/core/cairo_alpha/src/utils/version.ts

```diff
@@ -14,4 +14,4 @@ export const scarbVersion = '2.11.4';
 /**
  * Semantic version string representing of the minimum compatible version of Contracts to display in output.
  */
-export const compatibleContractsSemver = '2.0.0-alpha.1';
+export const compatibleContractsSemver = '^2.0.0-alpha.1';
```

Comment:
This was intended to be pinned to a specific version, since the alpha may not be forwards-compatible.
```suggestion
export const compatibleContractsSemver = '2.0.0-alpha.1';
```